### DISCUSSION TITLE
To do tax filter work with current query

### DISCUSSION THREAD

**Discussion by dregwide** [elementor/elementor#25217] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Create one product / post category template with loop grid and filters and assign it for all the categories instead of creating separate template for each. For big shops / resources its big difference create separately for each category or one for all. Nothing exist in nowadays without filtering. 

### Describe the solution you'd like.

Template, showing dynamically products / post of current category / page with filters.

### Describe alternatives you've considered.

None

### Additional context

Currently filter works with many query sources, but not with current query, wich is obviously huge lack, that causes alot of unnecessary work. With custom query its possible to create only 1 template for 1 separate page for content only on that page - dynamic content isn't present, which is very sad situation.
Elementor refers to https://github.com/orgs/elementor/discussions/23418 as to similar requests, but it is not similar.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by tehben** [comment]
I agree. I don't understand why this isn't a thing. It makes no sense that this isn't in the roadmap already. Why not improve compatibility with Woo rather than restrict it?

**Comment by katsuwoo-beep** [comment]
I agree.  I feel like this should be a core function, it shouldn't be necessary to shell out for an additional premium filter plugin.

**Comment by izakman** [comment]
Why doesn't the taxonomy filter widget work with the current query? This catches me out every time. There aren't even any notices/warnings or anything that make it clear it doesn't work.
